robotiq_gripper_gazebo_plugins/src/RobotiqGripperPlugin.cpp

```diff
@@ -114,6 +114,27 @@ namespace gazebo
 
     createGenericTopics(node_name);
 
+    if (_sdf->HasElement("p")){
```

Comment:
I would call this `kp`

Reply:
https://github.com/AcutronicRobotics/robotiq_modular_gripper/pull/2/commits/1e1314ecce4bf52483e366a87910af57a9017bd6